name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/dkb/DkbPDFExtractorTest.java

```diff
@@ -32,7 +32,6 @@
 import name.abuchen.portfolio.money.Money;
 import name.abuchen.portfolio.money.Values;
 
-@SuppressWarnings("nls")
```

Comment:
Gab es einen bestimmten Grund diese Annotation zu entfernen? Ich versuche das Projekt möglichst ohne Warnings in Eclipse zu haben. Und in dieser Testklasse gibt es ja keine zu übersetzenden Texte.

Bei mir sieht das übrigens so aus:
* Eine Meldung weil das Projekt gerade nicht im workspace ist (ist ein neuer branch)
* Eine Meldung über ein nicht übersetzes Label - stimmt - muss ich bei Gelegenheit fixen
* Und dann Meldungen in der Gruppe "Other" die von [SonarLint](https://sonarcloud.io/dashboard?id=name.abuchen.portfolio%3Aportfolio-app) Meldungen kommen - die ich aber nur "on-demand" fixe weil es auch einige false positives oder Geschmacksache sind

<img width="488" alt="Bildschirmfoto 2021-03-01 um 20 53 00" src="https://user-images.githubusercontent.com/587976/109550920-2eafee00-7ad0-11eb-8273-69e4396e0c2f.png">

Reply:
Hallo @buchen 
bei mir sieht das so aus...
![grafik](https://user-images.githubusercontent.com/45203494/109604753-59c42d00-7b24-11eb-83b1-70fef651653a.png)
![grafik](https://user-images.githubusercontent.com/45203494/109604796-6a74a300-7b24-11eb-8039-a879f8c2ae89.png)
![grafik](https://user-images.githubusercontent.com/45203494/109604889-92640680-7b24-11eb-8c2b-a33c1d4d21ca.png)

Habe ich eine falsche Einstellung?